Routes/Admin.js

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+import con from '../utils/db.js';
+import jwt from 'jsonwebtoken';
+
+const router = express.Router();
+
+router.post('/adminlogin', (req, res) => {
+    const sql = "SELECT * from admin where email = ? and password = ?";
+    con.query(sql, [req.body.email, req.body.password], (err, result) => {
+        if (err) return res.json({ loginStatus: false, Error: "Query error" });
+        if (result.length > 0) {
+            const email = result[0].email;
+            const token = jwt.sign({ role: "admin", email: email }, process.env.JWT_SECRET_KEY, { expiresIn: "1d"});
```

Comment:
create a utils file under under utils and add a function that will be responsible for generating tokens.. and just call the function here when you need the token